spec/ngsild_schema.json

```diff
@@ -0,0 +1,289 @@
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "enum": ["CRelationship"]
+                        },
+                        "object": {
+                          "type": "string",
+                          "format": "uri"
+                        }
+                    }
+                },
+                {
+                    "$ref": "#/definitions/CRelationshipSystemData"
+                },
+                {
+                    "$ref": "#/definitions/CRelationshipCommonData"
+                }
+            ]
+        },
+
+
+        "EntityInfo": {
+            "type": "object",
+            "required": ["id"],
+            "properties": {
+                "id": {
+                    "type": "string"
+                },
+                "idPattern": {
+                    "type": "string"
+                },
+                "type": {
+                    "type": "string"
+                }
+            }
+        },
+
+        "EntityInfo_correct_definition_not_compilable_in_swagger_2_0": {
+            "anyOf": [
+                {
+                    "type": "object",
+                    "required": ["id"],
+                    "properties": {
+                        "id": {
+                            "type": "string"
+                        }
+                    }
+                },
+                {
+                    "type": "object",
+                    "required": ["idPattern"],
+                    "properties": {
+                        "idPattern": {
+                            "type": "string"
+                        }
+                    }
+                },
+                {
+                    "type": "object",
+                    "required": ["type"],
+                    "properties": {
+                        "type": {
+                            "type": "string"
+                        }
+                    }
+                }
+            ]
+        },
+
+
+        "RegistrationInfo": {
+            "type": "object",
+            "required": ["entities"],
+            "properties": {
+                "entities": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "$ref": "#/definitions/EntityInfo"
+                    }
+                },
+                "properties": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "type": "string"
+                    },
+                    "uniqueItems": true
+                },
+                "relationships": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "type": "string"
+                    },
+                    "uniqueItems": true
+                }
+            }
+        },
+
```

Comment:
too many empty lines

Reply:
I was using a method to improve the speed reading code:
- Double line between main definitions (CProperty, CRelationship, RegistrationInfo, TimeInterval, ...)
- Single line between subdefinitions (CRelationshipCommonData, CRelationshipSystemData, CRelationship)

Fixed.